rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -66,13 +66,17 @@ class Pipeline(object):
     def __init__(self, batch_size=-1, num_threads=0, device_id=-1, seed=1,
                  exec_pipelined=True, prefetch_queue_depth=2,
                  exec_async=True, bytes_per_sample=0,
-                 rocal_cpu=False, max_streams=-1, default_cuda_stream_priority=0, tensor_layout=types.NCHW, reverse_channels=False, mean=None, std=None, tensor_dtype=types.FLOAT, output_memory_type=None):
+                 rocal_cpu=False, max_streams=-1, default_cuda_stream_priority=0, tensor_layout=types.NCHW, reverse_channels=False, mean=None, std=None, tensor_dtype=types.FLOAT, output_memory_type=None,
+                 last_batch_policy=types.LAST_BATCH_FILL, last_batch_padded=True):
+        if last_batch_padded is False:
+            print("last_batch_padded = False is not implemented in rocAL. Setting last_batch_padded to True")
+            last_batch_padded = True
         if (rocal_cpu):
             self._handle = b.rocalCreate(
-                batch_size, types.CPU, device_id, num_threads, prefetch_queue_depth, tensor_dtype)
+                batch_size, types.CPU, device_id, num_threads, prefetch_queue_depth, tensor_dtype, (last_batch_policy, last_batch_padded))
         else:
             self._handle = b.rocalCreate(
```

Comment:
I thought we are removing this from the context/pipeline parameters

Reply:
Removed the LastBatchPolicy from context/master_graph/Pipeline parameters and moved it to decoders.py.